Avoid copying default config into home directory during autogen.
Index: cava/autogen.sh
===================================================================
--- cava.orig/autogen.sh
+++ cava/autogen.sh
@@ -10,10 +10,3 @@ libtoolize
 aclocal
 autoconf
 automake --add-missing
-
-CONFIGDIR=$XDG_CONFIG_HOME/cava
-
-if [ -z "$XDG_CONFIG_HOME" ]; then CONFIGDIR=$HOME/.config/cava; fi
-
-mkdir -p "$CONFIGDIR"
-[ -f "$CONFIGDIR"/config ] || cp example_files/config "$CONFIGDIR"/config
